### ISSUE TITLE
Color styles don't seem to be working

### ISSUE THREAD

**Issue by mehtaculous** [recnepsbackwards/top_shot_sniper#1]
When I open the developer tools, I do see the color styles (with an `!important` tag) being applied to the correct serial numbers. However, it doesn't seem to be overriding the listing text, which is still showing up in white.

**Comment by recnepsbackwards** [comment]
Are you on PC or Mac?

**Comment by mehtaculous** [comment]
Mac

**Comment by recnepsbackwards** [comment]
This is a known issue for Mac. From my research it seems that styles cannot be applied to dropdowns. There could be some alternatives by adding HTML denotations such as a certain amount of * at the end of the option. Let me know if this is a desired feature and I can come up with something.

Or if you have a way to apply the styles to dropdowns properly.

**Comment by mehtaculous** [comment]
Definitely, I will try messing around with it and see if I can come up with a solution as well. Thanks for creating this!
